src/StoryBoard.cpp

```diff
@@ -465,8 +517,8 @@ bool StoryBoard::runFunct(unsigned int fID, NaughtyFilter &cm) {
 #endif
 
         if (isHeaderCheck) {
-            for (std::deque<String>::iterator u = cm.request_header->header.begin();
-                 u != cm.request_header->header.end(); u++) {
+            for (std::deque<String>::iterator u = targetheader->header.begin();
+                 u != targetheader->header.end(); u++) {
```

Comment:
Codacy found an issue: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238856)